lib/elixir/lib/option_parser.ex

```diff
@@ -114,6 +117,31 @@ defmodule OptionParser do
   end
 
   @doc """
+  The same as `parse/2` but raises an `ArgumentError` if it fails.
+  Returns the parsed values, remaining arguments and an empty list with the
+  invalid options otherwise.
```

Comment:
@alco Thanks for the feedback, I already made some changes as you suggested. Please, let me know what do you think.